packages/frontend/src/hooks/dashboard/useDashboardChart.ts

```diff
@@ -11,6 +12,7 @@ const useDashboardChart = (tileUuid: string, savedChartUuid: string | null) => {
 
     return useChartAndResults(
         savedChartUuid,
+        dashboardUuid ?? null,
```

Comment:
I guess it would be really unlikely for this to run without a `dashboardUuid`, right? 
should we update types in `postDashboardTile` to be `string | null`?

Reply:
Ignore, it's only `enabled` if it exists